cmd/embedded-cluster/preflights.go

```diff
@@ -150,13 +150,13 @@ var joinRunPreflightsCommand = &cli.Command{
 			return fmt.Errorf("embedded cluster version mismatch - this binary is version %q, but the cluster is running version %q", versions.Version, jcmd.EmbeddedClusterVersion)
 		}
 
-		setProxyEnv(jcmd.InstallationSpec.Proxy)
-		proxyOK, localIP, err := checkProxyConfigForLocalIP(jcmd.InstallationSpec.Proxy, c.String("network-interface"))
+		setProxyEnv(jcmd.InstallationSpec.RuntimeConfig.Proxy)
```

Comment:
revert all changes in this file back to using `jcmd.InstallationSpec.Proxy`

Reply:
done